src/components/WorkspaceFilter/WorkspaceFilter.js

```diff
@@ -164,48 +98,44 @@ const WorkspaceFilter = props => {
+                        <Button className='select-button' name='outlets' id={setProp(filter.outlets)} value={setProp(filter.outlets)} onClick={handleSelect}>Outlets Available</Button>
+                        <Button className='select-button' name='food' id={setProp(filter.food)} value={setProp(filter.food)} onClick={handleSelect}>Food</Button>
+                        <Button className='select-button' name='cowork' id={setProp(filter.cowork)} value={setProp(filter.cowork)} onClick={handleSelect}>Co-Work Space</Button>
+                        <Button className='select-button' name='coffee' id={setProp(filter.coffee)} value={setProp(filter.coffee)} onClick={handleSelect}>Coffee</Button>
+                        <Button className='select-button' name='alcohol' id={setProp(filter.alcohol)} value={setProp(filter.alcohol)} onClick={handleSelect}>Beer + Wine</Button >
+                        <Button className='select-button' name='openLate' id={setProp(filter.openLate)} value={setProp(filter.openLate)} onClick={handleSelect}>Open Late</Button >
+                        <Button className='select-button' name='lotsOfSeats' id={setProp(filter.lotsOfSeats)} value={setProp(filter.lotsOfSeats)} onClick={handleSelect}>Lots of seats</Button >
+                        <Button className='select-button' name='library' id={setProp(filter.library)} value={setProp(filter.library)} onClick={handleSelect}>Library</Button >
+                        <Button className='select-button' name='pets' id={setProp(filter.pets)} value={setProp(filter.pets)} onClick={handleSelect} name='pets'>Pet-Friendly</Button>
+                        <Button className='select-button' name='parking' id={setProp(filter.parking)} value={setProp(filter.parking)}  onClick={handleSelect}>Parking</Button>
+                        <Button className='select-button' name='goodForGroups' id={setProp(filter.goodForGroups)} value={setProp(filter.goodForGroups)} onClick={handleSelect}>Good for Groups</Button>
                       </div>
+
                       <div className='footer-div'>
                         <div>
                           <p onClick={clearFilter}>Clear Filters</p>
                         </div>
                         <Button className='submit-button' onClick={handleSubmit}>Apply Filters</Button>
                       </div>
+
                       <div className='rejection-message' id={rejection ? 'show' : 'hide'}>
                         <p>No matches found! Please apply different filters and try again.</p>
                       </div>
+
                     </Modal.Body>
                 </Modal>
               </div>
             }
 
             {matches.large &&
               <Fragment>
-                <Modal className='filter-modal' centered show={show} onHide={handleClose}>
+                <Modal className='filter-modal' centered show={show} onHide={clearFilter}>

```

Comment:
again, I think we want to maintain the state on close, so user can update filters.